client/header/activity-panel/unread-indicators.js

```diff
@@ -13,6 +13,15 @@ import { getSetting } from '@woocommerce/wc-admin-settings';
  */
 import { getUnreadNotesCount } from '../../inbox-panel/utils';
 
+const UNREAD_NOTES_QUERY = {
+	page: 1,
+	per_page: QUERY_DEFAULTS.pageSize,
+	status: 'unactioned',
+	type: QUERY_DEFAULTS.noteTypes,
+	orderby: 'date',
+	order: 'desc',
+};
```

Comment:
Moved this out, so that the variable `query` does not contain a new object reference every time (done in a couple other places as well).